tests/quantization/test_range_init.py

```diff
@@ -319,22 +331,32 @@ def __len__(self):
 
     act_quantizer_info = next(iter(compression_ctrl.non_weight_quantizers.values()))
 
-    def assert_range(quantizer: BaseQuantizer):
+    def check_scales(quantizer: BaseQuantizer, per_channel: bool):
         # Absolute tolerance is 1.0 due to percentile value interpolation
         if quantization_mode == 'symmetric':
-            assert quantizer.scale.item() == approx(6789, abs=1.0)
+            assert torch.allclose(quantizer.scale, torch.ones_like(quantizer.scale) * 6789, atol=1.0)
+            if per_channel:
+                assert quantizer.scale.numel() == 3
+            else:
+                assert quantizer.scale.numel() == 1
         else:
-            assert quantizer.input_low.item() == approx(3210, abs=1.0)
-            assert quantizer.input_range.item() == approx(3578, abs=1.0)
-
-    assert_range(act_quantizer_info.quantizer_module_ref)
+            assert torch.allclose(quantizer.input_low, torch.ones_like(quantizer.input_low) * 3210, atol=1.0)
+            assert torch.allclose(quantizer.input_range, torch.ones_like(quantizer.input_low) * 3578, atol=1.0)
+            if per_channel:
+                assert quantizer.input_low.numel() == 3
```

Comment:
you can avoid condition `if per_channel` by adding numel to params

Reply:
I think this way is more obvious. Would also complicate the parametrization iterable - can't get rid of *all* complexity here

Reply:
why not just use tuples ('symmetric', 1) and ('asymmetric, 3')?